src/main/flight/imu.c

```diff
@@ -345,31 +338,35 @@ static void imuMahonyAHRSupdate(float dt, float gx, float gy, float gz,
     gy *= (0.5f * dt);
     gz *= (0.5f * dt);
 
-    const float qa = q0;
-    const float qb = q1;
-    const float qc = q2;
-    q0 += (-qb * gx - qc * gy - q3 * gz);
-    q1 += (qa * gx + qc * gz - q3 * gy);
-    q2 += (qa * gy - qb * gz + q3 * gx);
-    q3 += (qa * gz + qb * gy - qc * gx);
+    const float qa = q.w;
```

Comment:
using a,b,c saved one char in old code (another possibility was q0t,q1t,..)
With quaternion structure, there is no namespace clash ...

But this is really minor detail ... 
(BTW: another possibility is to used 'delta' quaternion:

```
quaternion qDelta;
qDelta.w = (-q.x * gx - q.y * gy - q.z * gz);
...
q.w += qDelta.w;
...
```
Compiler shall be smart enough to optimize the code ...